### PR TITLE
Fix CountlyAnalytics NPE on MatrixClientPeg

### PR DESCRIPTION
**Pull request by t3chguy**
We convert to pseudonymous if user is opted in once we load the session later

### CODE REVIEW

File: src/components/structures/MatrixChat.tsx [hunk 1]
```diff
@@ -350,10 +350,8 @@ export default class MatrixChat extends React.PureComponent<IProps, IState> {
 
         if (SettingsStore.getValue("analyticsOptIn")) {
             Analytics.enable();
-            CountlyAnalytics.instance.enable(false);
```
Comment (jryans) [inline src/components/structures/MatrixChat.tsx:353]: Hmm, so was it backwards on purpose before...? (It used to say `false` when opted _in_...?)

Comment (t3chguy) [reply src/components/structures/MatrixChat.tsx:353]: the parameter is `anonymous=true` so when opted-in it'd say false, yes. But at this part of the app lifecycle we haven't yet got a MatrixClient so can't generate the pseudonymous countly userId yet

File: src/components/structures/MatrixChat.tsx [hunk 2]
```diff
@@ -350,10 +350,8 @@ export default class MatrixChat extends React.PureComponent<IProps, IState> {
...
-        } else {
-            CountlyAnalytics.instance.enable(true);
         }
+        CountlyAnalytics.instance.enable(true); // anonymous
```
Comment (jryans) [inline src/components/structures/MatrixChat.tsx:354]: Let's use the style we have elsewhere for annotating random booleans:

```suggestion
        CountlyAnalytics.instance.enable(/* anonymous = */ true);
```

Comment (t3chguy) [reply src/components/structures/MatrixChat.tsx:354]: image


Guess I'm spoiled :P

Comment (jryans) [reply src/components/structures/MatrixChat.tsx]: Ah yeah, my editor doesn't seem to do that at the moment 😜

**Review by jryans — APPROVED** [review]

**Pull request merged**